mlflow/tracking/_model_registry/utils.py

```diff
@@ -138,17 +138,26 @@ def _get_databricks_rest_store(store_uri, **_):
     return RestStore(partial(get_databricks_host_creds, store_uri))
 
 
-_model_registry_store_registry = ModelRegistryStoreRegistry()
-_model_registry_store_registry.register("databricks", _get_databricks_rest_store)
+_model_registry_store_registry = None
```

Comment:
nit: can we add a comment here explaining that this is an explicit change to avoid circular dependencies?